Ft current products

dreplica opened the pull request:
![Screenshot_20200426-221734_Expo](https://user-images.githubusercontent.com/27150392/80320087-3c3f7c00-880c-11ea-9b72-bc6ac65e75cf.jpg)
#### What does this PR do?

it displays current products to be selected by user

#### Description of Task to be completed?

Have the home screen show the following:

1. seperate cards showing an image for the product .
2. Name of the product is displayed.
3. Button to Add product and Leave product

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

#### Screenshots (if appropriate)

dreplica committed:
each button component stying is good

dreplica committed:
changed button height to applyscale

dreplica committed:
removed all padding from button to align element correctly, moved all height and weight element to textstyle, still no definite color for add and icon for leave empty

dreplica committed:
all items align properly, removed whitespace before +

dreplica committed:
included Styling creme & gels into regimen_setup.json

dreplica committed:
included a state to remove each product on clicking it

dreplica committed:
removed from remimen setup json = add atleast 3, included it in question setup, and reduced content width on currentposition

ogbodo commented:
This is how its displaying on my device.

Also I discovered when I click one any options there, it disappears... Is that on purpose?

![Simulator Screen Shot - iPhone Xs - 2020-04-26 at 22 39 40](https://user-images.githubusercontent.com/12932317/80320536-e5877180-880e-11ea-81dc-be9154430bfe.png)

dreplica commented:
> This is how its displaying on my device.
> 
> Also I discovered when I click one any options there, it disappears... Is that on purpose?
> 
> ![Simulator Screen Shot - iPhone Xs - 2020-04-26 at 22 39 40](https://user-images.githubusercontent.com/12932317/80320536-e5877180-880e-11ea-81dc-be9154430bfe.png)

-the button showing should be that disparity between ios and android. (I dont have an iphone to test the design)

-i did it to disappear just to make it  have an effect (since the  didnt specify).

ogbodo commented on src/screens/regimen_setup/currentProducts/index.tsx:
```diff
@@ -0,0 +1,73 @@
+            textStyle={{
+              width: applyScale(98),
+              height: applyScale(20),
+              color: '#BAB200',
```
you are suppose to move this color to the color file

dreplica replied:
thanks boss, I've moved it to theme

ogbodo commented on src/screens/regimen_setup/currentProducts/style.ts:
```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/native';
+import applyScale from '../../../utils/applyScale';
+
+export const Container = styled.View`
```
give this component flex 1 for screen optimisation

dreplica replied:
Changed it to flex:1 and removed width 100%

dreplica committed:
included save action without any action

dreplica committed:
implemented first phone verification countries.json and work files here

dreplica committed:
fixed font color for #BAB200 to FONT_DARK_GOLD_COLOR

dreplica committed:
changed container flex:1 and removed width:100%

ogbodo commented on src/screens/regimen_setup/currentProducts/index.tsx:
```diff
@@ -0,0 +1,81 @@
+}
+
+interface stateType {
+  display: 'flex' | 'none';
```
Please is this a type or a value? Am confused here bro

dreplica replied:
is a type to use with style{{display}}, style->display property accepts just that

dreplica replied:
but am supposed to delete this one to leave the later

ogbodo commented on src/screens/regimen_setup/currentProducts/index.tsx:
```diff
@@ -0,0 +1,81 @@
+
+export default function CurrentProduct(props: Iprops) {
+  const { colors, fonts } = useThemeContext();
+  const [state, setstate] = useState<'flex' | 'none'>('flex');
```
here too

dreplica replied:
Sorry for the duplication, this is the one am using, i declared a type for it earlier

dreplica committed:
Removed interface for logic

Removed the display type : 
interface stateType {
  display: 'flex' | 'none';
  added: ``;
}
